diff --git a/node_modules/@greenwood/plugin-import-jsx/src/index.js b/node_modules/@greenwood/plugin-import-jsx/src/index.js
index fa78471..da74b50 100644
--- a/node_modules/@greenwood/plugin-import-jsx/src/index.js
+++ b/node_modules/@greenwood/plugin-import-jsx/src/index.js
@@ -20,8 +20,7 @@ class ImportJsxResource {
 
     return (
       protocol === "file:" &&
-      this.extensions.includes(ext) &&
-      request.headers?.get("Accept").indexOf(this.contentType) >= 0
+      this.extensions.includes(ext)
     );
   }
 
