diff --git a/node_modules/@greenwood/cli/src/config/rollup.config.js b/node_modules/@greenwood/cli/src/config/rollup.config.js
index 5490610..119bcbc 100644
--- a/node_modules/@greenwood/cli/src/config/rollup.config.js
+++ b/node_modules/@greenwood/cli/src/config/rollup.config.js
@@ -514,7 +514,9 @@ function greenwoodSyncImportAttributes(compilation) {
               } else {
                 // waiting on Rollup to formally support `with`
                 // https://github.com/rollup/rollup/issues/5685
-                bundles[bundle].code = bundles[bundle].code.replace(/assert{/g, "with{");
+                // bundles[bundle].code = bundles[bundle].code.replace(/assert{/g, "with{");
+                // TODO: this is patched because for right now terser bundling is disable via a patch
+                bundles[bundle].code = bundles[bundle].code.replace(/assert {/g, "with {");
               }
 
               // check for app level assets, like say a shared theme.css
diff --git a/node_modules/@greenwood/cli/src/lib/layout-utils.js b/node_modules/@greenwood/cli/src/lib/layout-utils.js
index 94f4c34..36c3fae 100644
--- a/node_modules/@greenwood/cli/src/lib/layout-utils.js
+++ b/node_modules/@greenwood/cli/src/lib/layout-utils.js
@@ -157,12 +157,17 @@ async function mergeContentIntoLayout(
             return true;
           }
 
+          const contentType = "text/css";
           const resourceUrl = new URL(`file://${href}`);
-          const request = new Request(resourceUrl, { headers: { Accept: "text/css" } });
+          const request = new Request(resourceUrl, { headers: { Accept: contentType } });
           let isSupportedCustomFormat = false;
 
           for (const plugin of resourcePlugins) {
-            if (plugin.shouldServe && (await plugin.shouldServe(resourceUrl, request))) {
+            if (
+              plugin.contentType === contentType &&
+              plugin.shouldServe &&
+              (await plugin.shouldServe(resourceUrl, request))
+            ) {
               isSupportedCustomFormat = true;
               break;
             }
@@ -190,12 +195,17 @@ async function mergeContentIntoLayout(
             return true;
           }
 
+          const contentType = "text/javascript";
           const resourceUrl = new URL(`file://${src}`);
-          const request = new Request(resourceUrl, { headers: { Accept: "text/javascript" } });
+          const request = new Request(resourceUrl, { headers: { Accept: contentType } });
           let isSupportedCustomFormat = false;
 
           for (const plugin of resourcePlugins) {
-            if (plugin.shouldServe && (await plugin.shouldServe(resourceUrl, request))) {
+            if (
+              plugin.contentType === contentType &&
+              plugin.shouldServe &&
+              (await plugin.shouldServe(resourceUrl, request))
+            ) {
               isSupportedCustomFormat = true;
               break;
             }
diff --git a/node_modules/@greenwood/cli/src/plugins/resource/plugin-standard-javascript.js b/node_modules/@greenwood/cli/src/plugins/resource/plugin-standard-javascript.js
index 0f54727..dbf1ea8 100644
--- a/node_modules/@greenwood/cli/src/plugins/resource/plugin-standard-javascript.js
+++ b/node_modules/@greenwood/cli/src/plugins/resource/plugin-standard-javascript.js
@@ -78,7 +78,7 @@ const greenwoodPluginStandardJavascript = [
     type: "rollup",
     name: "plugin-standard-javascript:rollup",
     provider: (compilation) => {
-      return compilation.config.optimization !== "none" ? [terser()] : [];
+      return compilation.config.optimization !== "none" ? [] : [];
     },
   },
 ];
