diff --git a/node_modules/@greenwood/plugin-import-jsx/src/index.js b/node_modules/@greenwood/plugin-import-jsx/src/index.js
index 5fe6e11..f57e3a8 100644
--- a/node_modules/@greenwood/plugin-import-jsx/src/index.js
+++ b/node_modules/@greenwood/plugin-import-jsx/src/index.js
@@ -14,14 +14,14 @@ class ImportJsxResource {
     this.servePage = options.servePages ? "dynamic" : null;
   }
 
-  async shouldServe(url) {
+  async shouldServe(url, request) {
     const { pathname, protocol } = url;
     const ext = pathname.split(".").pop();
     
     return protocol === "file:" && this.extensions.includes(ext);
   }
 
-  async serve(url) {
+  async serve(url, request) {
     // refactor when WCC refactors
     // https://github.com/ProjectEvergreen/wcc/issues/116
     const tree = parseJsx(url);
